tools/runner/report.py

```diff
@@ -173,7 +175,7 @@ def group_results(data):
     Message is None if the test didn't produce a message"""
     tests = all_tests(data)
 
-    UAs = data.keys()
+    UAs = data.keys()  # noqa: N806
```

Comment:
Are these still necessary with the global noqa in this file?

If so, I think I'd prefer renaming the variables over sprinkling these comments.